src/modules/consumer.js

```diff
@@ -74,9 +80,20 @@ class Consumer {
 
       return this.channel.assertQueue(suffixedQueue, options).then((q) => {
         this._connection.config.transport.info(loggerAlias, 'init', q.queue);
+        this._connection.config.logger && this._connection.config.logger({
+          level: logLevels.Info,
```

Comment:
same here, think should be debug.

Reply:
I think pretty much all info logs in the repo should be debug, so I changed them.